### PR TITLE
1895 refactor gen pdf

### PR DESCRIPTION
**Pull request by matthopson**

### CODE REVIEW

**Review by sutt0n — APPROVED** [review]

**Review by codyseibert — APPROVED** [review]

File: web-client/src/presenter/actions/getFormValueDocumentAction.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/**
+ * gets the document key, value pair based on props
+ *
+ * @param {object} providers the providers object
+ * @param {object} providers.props the cerebral store used for getting the props.workItemId
```
Comment (kkoskelin) [inline web-client/src/presenter/actions/getFormValueDocumentAction.js:5]: needs updating (no `workItemId` here)

File: web-client/src/presenter/actions/getFormValueDocumentSizeAction.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+/**
+ * gets the document size key, value pair based on props
+ *
+ * @param {object} providers the providers object
+ * @param {object} providers.props the cerebral store used for getting the props.workItemId
```
Comment (kkoskelin) [inline web-client/src/presenter/actions/getFormValueDocumentSizeAction.js:5]: jsdoc update please

**Review by kkoskelin — COMMENTED** [review]
Just a few minor jsdoc tweaks -- but nice!

File: web-client/src/presenter/actions/getFormValueDocumentAction.js (hunk 1 above)
Comment (matthopson) [reply web-client/src/presenter/actions/getFormValueDocumentAction.js:5]: https://github.com/flexion/ef-cms/pull/2648

**Pull request merged**